app/assets/javascripts/setup.js

```diff
@@ -0,0 +1,6 @@
+$('.btn-group-toggle').click(function() {
+  $btnGroup = $(this);
```

Comment:
If you do that, you will declare `$btnGroup` as global (attached to window) and not in the function's scope.